### PR TITLE
Performance Improvements

### PR DESCRIPTION
**Pull request by lucasborin**

### CODE REVIEW

File: src/foundation/y_check_base.clas.abap [hunk 1]
```diff
@@ -711,11 +711,9 @@ CLASS Y_CHECK_BASE IMPLEMENTATION.
...
 
   METHOD get_class_description.
-    TRY.
-        result = NEW cl_oo_class( myname )->class-descript.
-      CATCH cx_class_not_existent.
-        result = 'Description Not Available'.
-    ENDTRY.
+    SELECT SINGLE descript INTO @DATA(description) FROM seoclasstx WHERE clsname = @myname.
+    result = COND #( WHEN description IS NOT INITIAL THEN description
+                     ELSE 'Description Not Available' ).
```
Comment (eugen-guenther-sap) [inline src/foundation/y_check_base.clas.abap:716]: you can write it that way:

```abap
SELECT SINGLE descript INTO @result FROM seoclasstx WHERE clsname = @myname.
IF sy-subrc <> 0.
result = 'Description Not Available'.
ENDIF.
```
If the description is avialble then it writes the entry into result.
if not, you get an subrc <> 0 and return the text.

**Review by eugen-guenther-sap — CHANGES_REQUESTED** [review]
see comments

File: src/foundation/y_check_base.clas.abap (hunk 1 above)
Comment (lucasborin) [reply src/foundation/y_check_base.clas.abap]: Done :)

**Comment by lucasborin** [comment]
@eugen-guenther-sap : Could you please review it again?

**Review by eugen-guenther-sap — APPROVED** [review]

**Pull request merged**